lib/streamlit/config.py

```diff
@@ -631,14 +631,6 @@ def _browser_server_port():
     expiration_date="2021-01-06",
 )
 
-_create_option(
-    "deprecation.showImageFormat",
-    description="Set to false to disable the deprecation warning for the image format parameter.",
-    default_val="True",
-    scriptable="True",
-    type_=bool,
-)
```

Comment:
We actually have an expiration param we can put on this so that if someone uses the config option it doesn't cry too hard for them and gives them some message about config removed I think. We should probably use that instead of deleting but ultimately up to product. cc @asaini

Reply:
We've had the deprecation warning for a while so I think it's safe to simply remove it (Our docs also reflect the the new param option)

Reply:
It wasn't clear what the resulting behavior from removing the config entirely and after testing, we'd like to instead deprecate the config option instead. @akrolsmir do you have time to address before Sunday 10:30pm PT so we can get this into the next release?

Reply:
I'll get it in right now.